src/main/java/seedu/financeit/utils/SaveManager.java

```diff
@@ -0,0 +1,133 @@
+package seedu.financeit.utils;
+
+import seedu.financeit.manualtracker.Ledger;
+import seedu.financeit.manualtracker.LedgerList;
+import seedu.financeit.manualtracker.ManualTracker;
+import seedu.financeit.manualtracker.subroutine.Entry;
+import seedu.financeit.manualtracker.subroutine.EntryList;
+import seedu.financeit.manualtracker.subroutine.EntryTracker;
+import seedu.financeit.parser.InputParser;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class SaveManager {
+    public static void save() throws IOException {
+        if(!Files.exists(Paths.get("./data"))){
+            Files.createDirectory(Paths.get("./data"));
+        }
+        if(!Files.exists(Paths.get("./data/save.txt"))){
+            Files.createFile(Paths.get("./data/save.txt"));
+        }
+        LedgerList LedList = ManualTracker.getLedgerList();
+        StringBuilder str = new StringBuilder();
+        int size = LedList.getItemsSize();
+
+        for (int i = 0; i < size; i++){
+            Ledger led = (Ledger) LedList.getItemFromIndex(i);
+            str.append(led.toString() + System.lineSeparator());
+            EntryList EntList = led.entryList;
+            int entsize = EntList.getItemsSize();
+            for (int x = 0; x < entsize; x++){
+                Entry ent = (Entry) EntList.getItemFromIndex(x);
+                str.append(ent.toString() + System.lineSeparator());
```

Comment:
So you used toString() to handle save text output.  You can perhaps introduce another method to the classes, possibly with an interface, to print some kind of save statement separately. This make make your life easier when you parse.